src/main/java/seedu/address/model/IngredientList.java

```diff
@@ -94,4 +94,8 @@ public CompatibleIngredientList getCompatibleIngredientList() {
     public ObservableList<Ingredient> getIngredientList() {
         return ingredients.asUnmodifiableObservableList();
     }
+
+    public void removeAllCartIngredients() {
+        ingredients.removeAllIngredients();
+    }
```

Comment:
It seems like this method should probably be in the `Cart` class rather that the `IngredientList` class which `Inventory` also extends from?

Reply:
Hmm, i think we could rename it and keep it in IngredientList, and maybe it could be repurposed?